liblangutil/ErrorReporter.cpp

```diff
@@ -28,6 +28,35 @@ using namespace std;
 using namespace dev;
 using namespace langutil;
 
+namespace langutil {
+
+std::string quote(std::string _name, bool _withSpaces)
+{
+	if (_withSpaces)
+		return " \"" + _name + "\" ";
+
+	return "\"" + _name + "\"";
+}
+
+}
+
+void checkMessage(string const _message)
+{
+	if (
+		_message.size() == 0 ||
+		_message[_message.size()-1] == '.' ||
```

Comment:
style

Reply:
actually multiple: binary operators have spaces around them and you should use `_message.back()`.

Reply:
`_message.back()` is certainly nicer. I don't understand the spaces part though? the `||` operators do have a space before the `or`ed conditions and we don't want a trailing spaces as far as I know

Reply:
It was about the `-`